app/config/config.yml

```diff
@@ -29,6 +29,8 @@ framework:
 twig:
     debug:            "%kernel.debug%"
     strict_variables: "%kernel.debug%"
+    paths:
+        "%kernel.root_dir%/../custom/email_templates": custom_email_templates
```

Comment:
Wow.  Thats annoying.  Yes we could have ilios do a post install composer hook where it checks out the files directory and creates learningmaterial and template files automatically.  In that case I think just leave it like this for now and we can make that change in another PR.